src/connectors/stats/__tests__/connectStats-test.js

```diff
@@ -33,6 +39,110 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/stats/js/#c
     });
   });
 
+  describe('getRenderState', () => {
+    test('returns the render state', () => {
+      const renderFn = jest.fn();
+      const unmountFn = jest.fn();
+      const createStats = connectStats(renderFn, unmountFn);
+      const stats = createStats();
+      const helper = jsHelper(createSearchClient(), 'indexName', {
+        index: 'indexName',
+      });
+      helper.search = jest.fn();
+
+      const renderState1 = stats.getRenderState(
+        { stats: {} },
+        createInitOptions({ helper })
+      );
+
+      expect(renderState1.stats).toEqual({
+        hitsPerPage: undefined,
+        nbHits: 0,
+        nbPages: 0,
+        page: 0,
+        processingTimeMS: -1,
```

Comment:
is this the current behaviour already for init? I wonder if it shouldn't be 0 instead; -1ms should never render I think

Reply:
if it's already the case, that would be a small breaking change / fix, but normally we don't render on init anyway

Reply:
`processingTimeMS` at `init` was already returning `-1`.
I think it make sense because returning `0` would mean that there was results, no?